tsdb/db.go

```diff
@@ -1238,6 +1246,14 @@ func (db *DB) compactHead(head *RangeHead) error {
 	return nil
 }
 
+func (db *DB) mmapHead() {
```

Comment:
This is a Head specific thing. We should avoid spilling that into db.go. And I don't think we should call it from here anywhere; Head should take care of calling it regularly.

Reply:
Moved to HEAD